__tests__/pattern/pattern.query.test.js

```diff
@@ -0,0 +1,349 @@
+    {
+      lang: 'uk',
+      value: testValue,
+    },
+    {
+      lang: 'en',
+      value: testValue,
+    },
+  ],
+  images: {
+    large: 'large_335nr4j5dkebkw5cy_test.jpg',
+    medium: 'medium_335nr4j5dkebkw5cy_test.jpg',
+    small: 'small_335nr4j5dkebkw5cy_test.jpg',
+    thumbnail: 'thumbnail_335nr4j5dkebkw5cy_test.jpg',
+  },
+  material: 'test',
+  handmade: false,
+  available: true,
+};
+
+beforeAll(async () => {
+  const res = await client
+    .mutate({
+      mutation: gql`
+        mutation($pattern: PatternInput!) {
+          addPattern(pattern: $pattern) {
+            ... on Pattern {
+              _id
+            }
+            ... on Error {
+              message
+              statusCode
+            }
+          }
+        }
+      `,
+
+      variables: { pattern: patternToAdd },
+    })
+    .catch(e => e);
+
+  patternId = res.data.addPattern._id;
+});
+afterAll(async () => {
+  await client
+    .mutate({
+      mutation: gql`
+        mutation($id: ID!) {
+          deletePattern(id: $id) {
+            ... on Pattern {
+              _id
+            }
+            ... on Error {
+              statusCode
+              message
+            }
+          }
+        }
+      `,
+      variables: { id: patternId },
+    })
+    .catch(e => e);
+});
+
+describe('pattern query tests', () => {
+  describe('tests for all patterns and one pattern', () => {
+    test('#1 Should receive all patterns', async () => {
+      const res = await client
+        .query({
+          query: gql`
+            query {
+              getAllPatterns {
+                items {
+                  name {
+                    lang
+                    value
+                  }
+                  description {
+                    lang
+                    value
+                  }
+                  images {
+                    large
+                    medium
+                    small
+                    thumbnail
+                  }
+                  material
+                  handmade
+                  available
+                }
+              }
+            }
+          `,
+        })
+        .catch(e => e);
+
+      expect(res.data.getAllPatterns).toMatchSnapshot();
+      expect(res.data.getAllPatterns).toBeDefined();
+      expect(res.data.getAllPatterns.items).toContainEqual({
+        __typename: 'Pattern',
+        name: patternToAdd.name.map(item => ({ ...languageTypeName, ...item })),
+        description: patternToAdd.description.map(item => ({
+          ...languageTypeName,
+          ...item,
+        })),
+        images: { ...imageTypeName, ...patternToAdd.images },
+        material: patternToAdd.material,
+        handmade: false,
+        available: true,
+      });
+    });
+    test('#2 Should receive one pattern', async () => {
+      try {
+        const res = await client
+          .query({
+            query: gql`
+              query($id: ID!) {
+                getPatternId(id: $id) {
```

Comment:
Please, rename the mutation to the `getPattern`

Reply:
actually it was merged not correct, resolver name is getPatternById. similarly like  other resolvers